tutorials/neow3j-nep17/index.md

```diff
@@ -0,0 +1,405 @@
+import io.neow3j.devpack.annotations.SupportedStandards;
+import io.neow3j.devpack.constants.CallFlags;
+import io.neow3j.devpack.constants.NativeContract;
+import io.neow3j.devpack.contracts.ContractManagement;
+import io.neow3j.devpack.events.Event3Args;
+```
+
+### Contract-specific Information
+
+Annotations on top of the contract's class represent contract-specific information. The following annotations can be used for a contract:
+
+_`@DisplayName`_
+
+Specifies the contract's name. If this annotation is not present, the class name is used for the contrat's name.
+
+_`@ManifestExtra`_
+
+Adds the provided key-value pair information in the manifest's `extra` field. You can also use `@ManifestsExtras` to gather multiple `@ManifestExtra` annotations (results in the same as when using single `@ManifestExtra` annotations).
+
+_`@SupportedStandards`_
+
+Sets the `supportedStandards` field in the manifest.
+
+_`@Permission`_
+
+Specifies, what other contracts and which methods it is allowed to call. By default (i.e., if no permission annotation is set), the contract is not allowed to call any contract. Use `contract = ` to specify what contracts and `methods = ` to specify what methods are allowed.
+
+_For example, if you want to allow transferring NEO tokens from the contract, you can add the annotation `@Permission(nativeContract = NativeContract.NeoToken, methods = "transfer")`._
+
+```java
+@DisplayName("AxLabsToken")
+@ManifestExtra(key = "author", value = "AxLabs")
+@SupportedStandards("NEP-17")
+@Permission(nativeContract = NativeContract.ContractManagement)
+public class FungibleToken {
+```
+
+### Constants
+
+You can set a constant value for the contract by using `final` variables. These values are always loaded when the contract is called and cannot be changed once the contract is deployed.
+
+> Note: The owner of this NEP-17 example contract is fixed (i.e., it is a `final` variable). If you intend to provide a functionality to change such a variable, you should not store it as a `final` variable. Rather, you would store it as a value in the storage, that you then can change through a method.
+
+-
+
+> Note: All contract constants and all methods have to be `static` (since the object-orientation of the JVM is different on the NeoVM).
+
+-
+
+```java
+    static final Hash160 owner = addressToScriptHash("NM7Aky765FG8NhhwtxjXRx7jEL1cnw7PBP");
+
+    static final int decimals = 2;
+    static final byte[] totalSupplyKey = new byte[]{0x00};
+    static final StorageContext ctx = Storage.getStorageContext();
+    static final StorageMap assetMap = new StorageMap(ctx, new byte[]{0x01});
+```
+
+### NEP-17 Methods
+
+The required NEP-17 methods are implemented in the following. If a method does not change the state of the contract (i.e., it is just used for reading), it can be annotated with the `@Safe` annotation. Out of the NEP-17 methods, only the `transfer` method should be writing to the contract and is thus not annotated as safe.
```

Comment:
```suggestion
The required NEP-17 methods are implemented as follows. If a method does not change the state of the contract (i.e., it is just used for reading), it can be annotated with the `@Safe` annotation. Out of the NEP-17 methods, only the `transfer` method should be writing to the contract and is thus not annotated as safe.
```